tensorflow_probability/python/distributions/two_piece_normal_test.py

```diff
@@ -348,6 +350,55 @@ def sampler(loc, scale, skewness):
       self.assertIsNotNone(grads[1])  # d/d scale
       self.assertIsNotNone(grads[2])  # d/d skewness
 
+  @test_util.numpy_disable_gradient_test
+  def testDifferentiableSampleNumerically(self):
+    """Test the gradients of the samples w.r.t. skewness."""
+    sample_shape = [int(2e5)]
+    seed = test_util.test_seed()
+
+    def sampler(skewness):
+      loc = tf.constant(0., self.dtype)
+      scale = tf.constant(1., self.dtype)
+      dist = tfd.TwoPieceNormal(
+          loc, scale=scale, skewness=skewness, validate_args=True)
+      return tf.reduce_mean(dist.sample(sample_shape, seed=seed))
```

Comment:
Can you add some non-linearity after the mean computation (tanh or something), to make sure this is checking the that the backwards pass gradients are correctly incorporated.

Reply:
As in `truncated_normal_test.testReparametrizable`, I used the nonlinear function `tf.abs` inside `tf.reduce_mean`. Is this OK?

Reply:
I think it should be okay. The goal is to get `dy` to not always be 1 and I think abs flips it to -1 sometimes?